ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/boot.j2

```diff
@@ -0,0 +1,27 @@
+{# System Boot Configuration #}
+{% if boot is arista.avd.defined %}
+
+# System Boot Settings
+{%     if boot.secret is arista.avd.defined %}
+
+## Boot Secret Summary
+{%         if boot.secret.sha512_password is arista.avd.defined %}
+{%             set method = "SHA512 ENCRYPTED" %}
+{%             set password = boot.secret.sha512_password %}
+{%         elif boot.secret.md5_password is arista.avd.defined %}
+{%             set method = "MD5 ENCRYPTED" %}
+{%             set password = boot.secret.md5_password %}
+{%         elif boot.secret.clear_text_password is arista.avd.defined %}
+{%             set method = "UNENCRYPTED" %}
+{%             set password = boot.secret.clear_text_password %}
+{%         endif %}
+
+- The {{ method }} Aboot password is: {{ password }}
```

Comment:
Let's not print out passwords in documentation. Just say that it is set, similar to enable password.